ingests_worker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_worker/IngestsWorkerFeatureTest.scala

```diff
@@ -0,0 +1,88 @@
+package uk.ac.wellcome.platform.storage.ingests_worker
+
+import org.scalatest.concurrent.{Eventually, IntegrationPatience}
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import uk.ac.wellcome.json.JsonUtil._
+import uk.ac.wellcome.messaging.fixtures.SQS.QueuePair
+import uk.ac.wellcome.platform.archive.common.fixtures.HttpFixtures
+import uk.ac.wellcome.platform.archive.common.generators.IngestGenerators
+import uk.ac.wellcome.platform.archive.common.ingests.models.IngestUpdate
+import uk.ac.wellcome.platform.storage.ingests_worker.fixtures.IngestsWorkerFixtures
+
+class IngestsWorkerFeatureTest
+    extends AnyFunSpec
+    with Matchers
+    with Eventually
+    with HttpFixtures
+    with IngestsWorkerFixtures
+    with IngestGenerators
+    with IntegrationPatience {
+
+  val visibilityTimeoutInSeconds = 1
+  val waitTimeInMilliseconds = (visibilityTimeoutInSeconds + 1) * 1000
+
+  it("When the client succeeds it should consume the message") {
```

Comment:
tiny nit: we've shied away from "should" in tests:

```suggestion
  it("when the client succeeds, it consumes the message") {
```